distance-calculator/visualization/dashboard.json

```diff
@@ -249,16 +259,16 @@
       }
     }
   ],
-  "refresh": false,
-  "schemaVersion": 22,
+  "refresh": "5s",
+  "schemaVersion": 27,
   "style": "dark",
   "tags": [],
   "templating": {
     "list": []
   },
   "time": {
-    "from": "now-5m",
-    "to": "now"
+    "from": "2022-11-08T10:00:00.000Z",
+    "to": "2022-11-08T10:59:59.000Z"
```

Comment:
`from`, `to` may better not set to some sepecfic time. may refer to previous values as an example.

Reply:
Reverted to older properties.